### ISSUE TITLE
Pluto

### ISSUE THREAD

**Issue by Tusay** [PsuAstro528/project-mo9899#11]
https://github.com/PsuAstro528/project-mo9899/blob/ff127f92b5b24fe9727ce23213aa454aac2d3d81/Spectral%20Fitting.jl#L1

Did you choose to code this for Pluto out of preference or convenience or just because we had done that for all the labs previously? Have you considered compressing it into a standalone .jl file without all the Pluto comments and things? It may be easier to review and work on in some ways (and possibly more difficult in others). 

I coded our project outside of Pluto in an empty .jl file. I find it makes the whole thing significantly shorter and easier to manage. The markdowns and inline plots in Pluto are nice though.

**Comment by eford** [comment]
Good point.  I'd suggest moving the core project code into one or more files in the src directory.  Ideally, you'd make them into a module, so they can be precompiled.  Then you make a Pluto or Jupyter notebooks that loads that code to perform demonstrations, tests, and benchmarks.
